### PR TITLE
Multilingual update with first complete translation (Japanese)

### PR DESCRIPTION
**Pull request by emergent-reasons**

### CODE REVIEW

**Comment by emergent-reasons** [comment]
Can't add for formal review so pinging you here: @zquestz

File: src/i18n/ja.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      title: クライアントソフトウェア
+      description: CashShuffleのリファレンス実装は、人気のElectron Cashウォレットから利用可能です。
+    server:
+      title: サーバーソフトウェア
+      description: サーバーサイドソフトウェアはGolangで書かれておりGitHubで利用可能です。
+    docs:
+      title: ドキュメンテーション
+      description: 開発者および愛好家のための技術文書
+  home:
+    title: CashShuffle  -  Bitcoin Cashのためのプライバシー機能
```
Comment (zquestz) [inline src/i18n/ja.yml]: NIT: Don't need 2 spaces here. =)

File: src/i18n/ja.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    code: ja
+    display: 日本語
+    ja: true
+  nav:
+    home: ホーム
+    wallets: ウォレット
+    quickstart: クイックスタート
+    faqs: よくある質問
+  footer:
+    copyright: Copyright &copy; 2017-2019 CashShuffle. All rights reserved.
```
Comment (zquestz) [inline src/i18n/ja.yml:15]: NIT: Shouldn't this be translated?

**Comment by zquestz** [comment]
This looks great, just a couple nits. Cameron did a great job. We should focus on getting more quality translations too! Going to let a few of the other reviewers take a look before I merge.

File: .build/ja/index.html [hunk 3]
```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="ja" prefix="og: http://ogp.me/ns#">
+  <head>
+    <meta charset="utf-8">
+    <title>CashShuffle  -  Bitcoin Cashのためのプライバシー</title>
```
Comment (zquestz) [inline .build/ja/index.html]: Yeah a few places with these double spaces. Probably should normalize them.

File: src/i18n/ja.yml (hunk 1 above)
Comment (emergent-reasons) [reply src/i18n/ja.yml]: Checking with translator. Also I fixed a bunch of whitespace at the end of lines.

Comment (emergent-reasons) [reply src/i18n/ja.yml]: Ok I fixed all the whitespace at the end of lines and also removed those double spaces you found. Translator did not intend that.

dbe24e2

File: src/i18n/ja.yml (hunk 2 above)
Comment (emergent-reasons) [reply src/i18n/ja.yml:15]: Translator says at least for Japanese it's better to leave this in English. He intentionally changed it back from the machine translation.

File: src/i18n/ja.yml (hunk 1 above)
Comment (emergent-reasons) [reply src/i18n/ja.yml]: I pushed to the wrong branch. Hold on.

Comment (emergent-reasons) [reply src/i18n/ja.yml]: 4ef041c

File: .build/ja/index.html (hunk 3 above)
Comment (emergent-reasons) [reply .build/ja/index.html]: Wait. Why is the build in the repo?

File: src/i18n/ja.yml (hunk 1 above)
Comment (emergent-reasons) [reply src/i18n/ja.yml]: Also in the original english: 592c8ff

File: .build/ja/index.html (hunk 3 above)
Comment (emergent-reasons) [reply .build/ja/index.html]: Thanks for explaining offline. I rebuilt for production `grunt live` and `rebuild.sh -p` and pushed the changes.

**Pull request merged**